src/extensions/extensionsApi.ts

```diff
@@ -476,7 +476,7 @@ export async function listExtensionVersions(
     }>(`/publishers/${publisherId}/extensions/${extensionId}/versions`, {
       queryParams: {
         filter,
-        show_prereleases: String(show_prereleases),
+        showPrereleases: String(showPrereleases),
```

Comment:
```suggestion
        show_prereleases: String(showPrereleases),
```

Reply:
Does the backend accept either casing of the query parameter? If not, we should revert. (I'm guessing either works if it is a One Platform API, but we may want to play safe)